### PR TITLE
adding make targets to run auraed as pid 1 within a libvirt VM

### PR DESCRIPTION
**Pull request by MalteJ**
Hi Kris,

together with @vincinator I am working on a project somewhat similar to aurae. We are writing a pid 1 in Rust to act as a VMM (basically a replacement for systemd+libvirt), that exposes a gRPC API to control VMs from a SmartNIC. It's not published yet, but will also be under Apache v2.

I have extracted our tooling around testing the program as pid 1 within a libvirt VM.
Have a look at `hack/README.md` for details.

Basically this PR allows you to start a VM, boot a Kernel and load auraed as pid 1. With the current state of auraed you see a Kernel panic as auraed expects things on the Linux system, that don't exist (e.g. syslog).

We can discuss the way those scripts are implemented. I am open for changes.

Malte

### CODE REVIEW

**Comment by krisnova** [comment]
Hello and thank you for a very wonderful contribution. If you can please sign the CLA I will merge this feature and it will become a core part of the project. Thank you again for your contribution!

Please [sign the Aurae CLA](https://docs.google.com/forms/d/e/1FAIpQLSfoBdAs4duoL96q49hioCuAm3c0ClKTzsSdlBgvbVi7GAmzdQ/viewform) and I will merge!

**Comment by MalteJ** [comment]
@cla-bot recheck plz

**Comment by MalteJ** [comment]
is there a magic word that makes the @cla-bot check again?

**Comment by MalteJ** [comment]
@kris-nova I have signed the CLA, but the bot still returns an error. Can you please have a look?

**Comment by krisnova** [comment]
@cla-bot recheck

**Comment by MalteJ** [comment]
@cla-bot check

I think it's `check` not `recheck`

**Comment by krisnova** [comment]
Welcome to Aurae @MalteJ 

We are in Discord if you want to talk to us further.
